CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ✨ You do not need to add a pull request reference or author information, this will be done automatically by CI. ✨
 -->
 
+- Added an option to pass the web token as X-Token header
+
```

Comment:
Please move this into the unreleased section.

Reply:
Done in f02eb9d48a326853b56c7a31c605de45a660ca5c